### DISCUSSION TITLE
Dicom file conversion to png failure

### DISCUSSION THREAD

**Discussion by wh75er** [ImageMagick/ImageMagick#3432] [Help]
Hello, I'm using ImageMagick 7.0.10-58 Q16 x86_64 2021-01-16

And I'm trying to convert dicome image to png. I'm not really sure how to do it, but there's some strange with my file. If I use `identify` on it, I'm getting null output even with verbose:

```
identify -verbose IM0001
```

But `file` gives this output:

```
file IM0001
IM0001: DICOM medical imaging data
```

When `convert`:

```
convert IM0001 im.png
convert: no images defined `im.png' @ error/convert.c/ConvertImageCommand/3301.
```

Does anyone know what's wrong and how to fix that?

**Comment by dlemstra** [comment]
Is it possible to anonymize the file and upload it here or share it trough email? And can you reproduce this issue with a more recent version?

**Reply by wh75er** [reply]
Yeah, sure. I reproduced this issue on `Version: ImageMagick 7.0.11-4 Q16 x86_64 2021-03-20`. Output is the same

I've sended you this file via email. Email's subject is `Dicom file conversion to png failure #3432`

**Reply by dlemstra** [reply]
We are unable to read your image. In the next version of ImageMagick an exception will be thrown to make it clear that this is happening.

**Reply by wh75er** [reply]
Could you be more specific? What exactly wrong with the image so I could fix that somehow...

**Reply by dlemstra** [reply]
While reading the file it fails at some point. I cannot help you with debugging why it fails.
